File: PHP/8php 2/monkeyCount.php
<?php
/*
|--------------------------------------------------------------------------
| Count the Monkeys!
|--------------------------------------------------------------------------
|
*/

function monkeyCount($n) {
   $res = [];
   for ($i=1; $i <= $n; $i++) { 
      array_push($res, $i);
   }
   return $res;
}


function monkeyCount($n) {
   return range(1, $n);
}


function monkeyCount($n) {
   $counter = [];
   while ($n > 0){
      $counter[] = $n;
      $n--;
   }
   return array_reverse($counter);
}


function monkeyCount($n) {
   return array_fill(1, $n,range(1, $n))[1];
}


print_r(monkeyCount(5));
print_r(monkeyCount(3));

File: Python/8py 3/monkey_count.py
"""
|--------------------------------------------------------------------------
| Count the Monkeys!
|--------------------------------------------------------------------------
|
"""

def monkey_count(n):
   res = [];
   for i in range(1, n+1):
      res.append(i)
   return res


monkey_count = lambda n: list(range(1, n+1))


def monkey_count(n):
   return range(1, n+1)


def monkey_count(n):
   return [x for x in range(1,n+1)]


def monkey_count(n):
   L=[]
   L.extend(range(1,n+1))
   return(L)


print(monkey_count(5), [1, 2, 3, 4, 5])
print(monkey_count(3), [1, 2, 3])